hirzenbach-bot/main.py

```diff
@@ -12,7 +12,7 @@
 from threading import Thread
 import gpt3
 from env import Env
-from persistance import Data
+from persistence import init_database
```

Comment:
can't we delete `persistance.py` now? Apologies if you already did and I misinterpreted the UI.

Reply:
Persist**a**nce got renamed to Persist**e**nce (the diff shows it as renamed to migrate.py)